flask_saml_sso/sso.py

```diff
@@ -74,16 +80,27 @@ def _get_saml_settings(app):
     s.setdefault('idp', {}).update(remote.get('idp'))
 
     if requests_signed:
-        with open(cert_file, 'r') as cf:
-            cert = cf.read()
-        with open(key_file, 'r') as kf:
-            key = kf.read()
+        try:
+            with open(cert_file, 'r') as cf:
+                cert = cf.read()
+        except OSError:
+            logger.error('Unable to read cert file {}'.format(cert_file))
```

Comment:
Include exception info as well?
```suggestion
            logger.exception('Unable to read cert file {}'.format(cert_file))
```